jetstream/whats-new-notification-sidebarvertical-tabs.toml

```
@@ -0,0 +1,34 @@
+
+[experiment]
+
+#segment information:https://docs.telemetry.mozilla.org/concepts/segments.html
+segments = ["clients_last_seen.is_new_or_resurrected_v3", "clients_last_seen.activity_segments_v1"]
+
+
+[metrics]
+# Metrics to compute for each weekly analysis window.
+weekly = ["notification_clicks","avg_max_concurrent_vertical_tab_count"]
+
+# Metrics to compute for each 28 days long analysis window
+28_day = ["notification_clicks","avg_max_concurrent_vertical_tab_count"]
+
+# Metrics to compute only for the overall analysis window.
+overall = ["notification_clicks","avg_max_concurrent_vertical_tab_count"]
+
+[metrics.avg_max_concurrent_vertical_tab_count]
+#averaging the count of maximum number of tabs with vertical tabs enabled per client_id, branch over an analysis window
+#probe:https://probes.telemetry.mozilla.org/?search=max_concurrent_vertical_tab_count&view=detail&probeId=scalar%2Fbrowser.engagement.max_concurrent_vertical_tab_count
+select_expression= "AVG(COALESCE(payload.processes.parent.scalars.browser_engagement_max_concurrent_vertical_tab_countwser_engagement_max_concurrent_vertical_tab_count,0))"
```

Updating the metric definition as @florezita @bani recommended. 
```suggestion
select_expression= "CAST(SUM(COALESCE(payload.processes.parent.scalars.browser_engagement_max_concurrent_vertical_tab_count ,0)) > 0 AS INT)"
```